core-services/image-mirroring/openshift/mapping_origin_4_8

```diff
@@ -129,7 +129,6 @@ registry.ci.openshift.org/origin/4.8:kubefed-operator quay.io/openshift/origin-k
 registry.ci.openshift.org/origin/4.8:kubemark-machine-controllers quay.io/openshift/origin-kubemark-machine-controllers:4.8 quay.io/openshift/origin-kubemark-machine-controllers:4.8.0
 registry.ci.openshift.org/origin/4.8:kubernetes-tests quay.io/openshift/origin-kubernetes-tests:4.8 quay.io/openshift/origin-kubernetes-tests:4.8.0
 registry.ci.openshift.org/origin/4.8:kubevirt-csi-driver quay.io/openshift/origin-kubevirt-csi-driver:4.8 quay.io/openshift/origin-kubevirt-csi-driver:4.8.0
-registry.ci.openshift.org/origin/4.8:kubevirt-must-gather quay.io/openshift/origin-kubevirt-must-gather:4.8 quay.io/openshift/origin-kubevirt-must-gather:4.8.0
```

Comment:
why is this being removed?

Reply:
> why is this being removed?

As far as I understand it, this is not in use at all. @tiraboschi - WDYT?

Reply:
well it's being published, it was published as recently as 37 minutes ago.  So it's hard to say "no one is using it".

assuming it's not fundamentally broken in some way, i don't see why we wouldn't continue to publish it.

that said, apparently we never published a 4.9/4.10/4.11 tag.

so maybe my real question should be, why aren't we publishing it generally?

(it looks like we published a 4.6 tag, and a 4.8 tag, only)

Reply:
Not sure about the history of it. But the formal release is done elsewhere. @tiraboschi - do you agree?

Reply:
yes, we are pushing to quay.io/repository/kubevirt/must-gather (without tagging),
I don't think we are using quay.io/openshift/origin-kubevirt-must-gather:4.8